### PR TITLE
Working branch

### PR DESCRIPTION
**Pull request by Maryjanee**
### Project Description

- Implemented the bubble sort algorithm by creating a method called bubble_sort
- Implemented the bubble sort by algorithm by creating a method called bubble_sort_by using yield.

### CODE REVIEW

**Review by kobiyoyo — CHANGES_REQUESTED** [review]
## Hi Team     
#### Great Job so far, but there are a few more things required. 
### General
- [x] Let's work on the readme.MD file (The usage section).
- [x] The methods `bubble_sort` & `bubble_sort_by`  should be in one file 
### Make changes and submit another code review request.
*Happy coding* 
(Any questions Feel free to contact me on slack @kobiyoyo)

File: README.md [hunk 1]
```diff
@@ -1 +1,71 @@
-# Bubble-Sort
\ No newline at end of file
+# Bubble Sort
+
+This project consist of implementing the bubble sort algorithm with the ruby programming language.
```
Comment (santiago-dev96) [inline README.md]: Proper names are capitalized.
```suggestion
This project consists of implementing the bubble sort algorithm with the Ruby programming language. 
```

File: README.md [hunk 2]
```diff
@@ -1 +1,71 @@
...
\ No newline at end of file
+# Bubble Sort
+
+This project consist of implementing the bubble sort algorithm with the ruby programming language. 
+
+
+## Built With
+
+- Ruby
+- Rubocop
```
Comment (santiago-dev96) [inline README.md]: This is not necessary.
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -1 +1,71 @@
-# Bubble-Sort
\ No newline at end of file
+# Bubble Sort
+
+This project consist of implementing the bubble sort algorithm with the ruby programming language. 
+
+
+## Built With
```
Comment (santiago-dev96) [inline README.md]: I suggest changing this to something more clear.
```suggestion
## Requirements
```

File: README.md [hunk 4]
```diff
@@ -1 +1,71 @@
...
+
+- Ruby
```
Comment (santiago-dev96) [inline README.md]: Try to specify the Ruby version you used for this project.
```suggestion
- Ruby (X.X.X)
```

File: README.md [hunk 5]
```diff
@@ -1 +1,71 @@
...
+
+[Live Demo Link](https://repl.it/@MaryjaneA/DazzlingPolishedIdentifier#main.rb)
+​
+
+## Getting Started
+
+To get a local copy of the repository please run the following commands on your terminal:
+
+$ cd <folder>
+$ git clone git@github.com:Maryjanee/Bubble-Sort.git
```
Comment (santiago-dev96) [inline README.md]: That's not the way you provide directions to issue commands on the terminal. Also, other users can't clone your repositories using SSH. Surround the following with three backticks and at the end of the opening backticks type **sh**.
```suggestion
git clone https://github.com/Maryjanee/Bubble-Sort.git
cd Bubble-Sort
```

**Review by santiago-dev96 — APPROVED** [review]
# Project approved! :100:

Everything works fine, but your `README.md` still needs some work, I'll suggest implementing the changes requested as soon as possible, this is optional though, just some details.

I wish you the best for your next projects! 😄

**Pull request merged**